docs/intro/tutorial03.txt

```diff
@@ -121,9 +121,10 @@ like so::
 
 The ``question_id=34`` part comes from ``<int:question_id>``. Using angle
 brackets "captures" part of the URL and sends it as a keyword argument to the
-view function. The ``:question_id>`` part of the string defines the name that
-will be used to identify the matched pattern, and the ``<int:`` part is a
+view function. The ``question_id>`` part of the string defines the name that
+will be used to identify the matched pattern, and the ``<int`` part is a
```

Comment:
I think we can remove the angle brackets two as they're already discussed and I think it is wrong to include them.
```suggestion
view function. The ``question_id`` part of the string defines the name that
will be used to identify the matched pattern, and the ``int`` part is a
```